tools/cgmanifest.json

```diff
@@ -385,7 +386,7 @@
         "Type": "nuget",
         "Nuget": {
           "Name": "runtime.native.System.Data.SqlClient.sni",
-          "Version": "4.7.0"
+          "Version": "4.4.0"
```

Comment:
"4.7.0" -> "4.4.0" ?

Reply:
Why do we reference this package here, it is a dependancy of `System.Data.SqlClient`.